vm/message/src/signed_message.rs

```diff
@@ -25,6 +25,9 @@ impl SignedMessage {
         Ok(SignedMessage { message, signature })
     }
 
+    pub fn new_from_fields(message: UnsignedMessage, signature: Signature) -> Self {
```

Comment:
idk optional but maybe `new_from_parts` is more in line with naming standards, from fields is a bit confusing in this context